### ISSUE TITLE
Support for :except option

### ISSUE THREAD

**Issue by chap** [fnando/recurrence#14]
(First off, love this library, it has saved us a ton of time.)

We're doing some integration with icalendars, and they allow reoccurring events to specify dates NOT to occur (They call it [EXDATE](http://www.kanzaki.com/docs/ical/exdate.html)).

I might be missing something, but it seems like this feature is not supported yet.

Since we'll have to write it, how do you guys feel about a syntax like this:

``` ruby
recurrence(:every => :day, :except =>  "2010-11-20")
recurrence(:every => :day, :except =>  ["2010-11-20", "2010-11-21"])
```

**Comment by oliamb** [comment]
Hi,

I am very interested by this feature, what is the conclusion of this issue? Implemented or abandoned?

Cheers

**Comment by chap** [comment]
We've been using it in production for a few years!

**Comment by chap** [comment]
Looks like it got merged in v1.3.0 https://github.com/fnando/recurrence/pull/15
